cluster/aws/coreos/util.sh

```diff
@@ -30,8 +30,8 @@ function detect-minion-image (){
 
 function generate-minion-user-data() {
   i=$1
+  # TODO(bakins): Is this actually used?
```

Comment:
I do not see it being used anywhere. I guess I didn't when I wrote it either, so not sure why I even have it there.